ui_src/src/components/tagsPicker/style.scss

```diff
@@ -0,0 +1,135 @@
+.tags-picker-wrapper {
+    width: 100%;
+    align-items: center;
+    align-content: center;
+    overflow-x: hidden;
+    .tags-picker-title {
+        margin-left: 12%;
+        font-family: 'Inter';
```

Comment:
The default font in our system is Inter, so you don't need to define it again.
Please check where you define it and delete it.